packages/opensrp-store/src/store/ducks/index.ts

```diff
@@ -0,0 +1,223 @@
+/** users ducks modules: actions, actionCreators, reducer and selectors */
+import { Dictionary } from '@onaio/utils';
+import { keyBy, values } from 'lodash';
+import intersect from 'fast_array_intersect';
+import { AnyAction, Store } from 'redux';
+import { createSelector, OutputParametricSelector } from 'reselect';
+import SeamlessImmutable from 'seamless-immutable';
+
+/** The reducer name */
+export const reducerName = 'keycloakUsers';
```

Comment:
This reducer module needs to be moved to the keycloak user package

Reply:
https://github.com/OpenSRP/web/issues/64